### ISSUE TITLE
Default keyboard: Focus indicator is not visible on the "Search" button

### ISSUE THREAD

**Issue by Nina-1210** [microsoft/pxt-microbit#6826]
**Describe the bug**
When navigating using the default keyboard (Tab key), the focus indicator (focus border) is not visible when focus moves to the "Search" button. 

**Steps to reproduce the behavior:**
1. Navigate to https://makecode.microbit.org/beta.
2. Use the **Tab** key to move focus through interactive elements.
3. Continue tabbing until the focus reaches the "**Search**" button.
4. Observe the visual focus indicator.

**Actual behavior**
No visible focus indicator (e.g., border or outline) is displayed when the "Search" button receives keyboard focus.
 
**Expected behavior**
A clearly visible focus indicator (e.g., outline or border) should be displayed when the "Search" button is focused, ensuring proper keyboard navigation feedback and compliance with accessibility standards (e.g., WCAG).
 
**Screen Recording**
https://github.com/user-attachments/assets/08baeeb1-a328-4f8a-bc9b-c441ab203fc8

**Additional context**
1.OS: Windows 11 25H2
2.Browser: Chrome, Edge, and Firefox
3.makecode.microbit.org version: 8.1.15
4.Microsoft MakeCode version: 12.3.10

**Comment by Nina-1210** [comment]
Verified the bug on latest beta build: **8.1.17**, it has been fixed. 

<img width="882" height="498" alt="Image" src="https://github.com/user-attachments/assets/9c007809-ced9-468a-8133-40477268aa00" />
